device-types/Ubiquiti/U6-Lite.yaml

```diff
@@ -1,10 +1,21 @@
 ---
 manufacturer: Ubiquiti
-model: U6-Lite
-slug: u6-lite
+model: Access Point U6 Lite
+part_number: U6-Lite
+slug: ubiquiti-u6-lite
 u_height: 0
 is_full_depth: false
-comments: '[UniFi 6 Lite Access Point](https://store.ui.com/products/unifi-ap-6-lite)'
+comments: |
+  [UniFi 6 Lite Access Point](https://store.ui.com/products/unifi-ap-6-lite)
+airflow: passive
+weight: 300.0
+weight_unit: g
 interfaces:
   - name: lan0
     type: 1000base-t
+    poe_mode: pd
+    poe_type: type1-ieee802.3af
+  - name: wlan0
+    type: ieee802.11n
```

Comment:
Where are you getting `n` from? This is an ax access point

Reply:
```suggestion
    type: ieee802.11x
```